modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb

```diff
@@ -126,6 +126,8 @@ def exploit
     ms16_032.gsub!("$cmd","\"#{cmdstr}\"")
     #lpcommandLine - capped at 1024b
     ms16_032.gsub!("$args1","\"#{psh_cmd}\"")
+    end_flag = Rex::Text.rand_text_alphanumeric(32)
+    ms16_032.gsub!("$end","\"#{end_flag}\"")
```

Comment:
Does this need an extra set of escaped quotes with `$end` quoted in the powershell script?

Reply:
You're right, they're not needed. Thanks!